sales/send_quotations/terms_and_conditions.rst

```diff
@@ -2,43 +2,98 @@
+
+In the settings of the **Invoicing App**, you will have the possibility to insert your default terms
+and conditions.
+
+.. image:: media/terms_conditions_2.png
+   :align: center
+   :class: img-thumbnail
+   :alt: Default terms & conditions on quotation on Odoo Sales?
+
+They will appear subsequently on every quotation, sales order and invoice.
+
+Add terms & conditions to your quotation templates
+==================================================
+
+For even more convenience, you can specify your terms & conditions on every quotation templates
+that you create. These conditions will only appear if you choose to use a distinct quotation
+template when creating a quotation.
 
-.. image:: media/terms_and_conditions02.png 
-  :align: center 
+.. image:: media/terms_conditions_3.png
+   :align: center
+   :class: img-thumbnail
+   :alt: Add terms & conditions to your quotation templates on Odoo Sales?
 
-.. image:: media/terms_and_conditions03.png
-  :align: center
+.. important::
+   Be sure to check out our documentation on how to :doc:`quote_template` to master each step of
+   this amazing feature.
   
-Set up more detailed terms & conditions
-=======================================
+Add more detailed terms & conditions
+====================================
+
+Here are different ideas to share more details about your terms & conditions with your customers:
+
+- Use the **Website App** and create your own terms & conditions page to which you can refer.
+  For example, here is the Odoo terms & conditions page that we send to our customers:
+
+.. image:: media/terms_conditions_4.png
+   :align: center
+   :class: img-thumbnail
+   :alt: Terms & conditions on your website?
+
+- Attach an external document with more detailed conditions when you are about to send your
+  quotation by email to your customers.
+
+.. image:: media/terms_conditions_5.png
+   :align: center
+   :class: img-thumbnail
+   :alt: Terms & conditions in attachments in your email?
+
+- Create and edit email templates to set a default attachment for all quotation emails that you
+  will send in the future. To do so, you have to go to :menuselection:`Sales --> Configuration
+  --> Quotation templates` and create a new quotation template or modify an existing one. You
+  will see that under the confirmation tab, you will be able to activate online signatures,
+  online payments and to set a confirmation mail in which you will have the possibility to configure
+  the default attachment. There, you can put your detailed terms & conditions.
+
+.. image:: media/terms_conditions_6.png
+   :align: center
+   :class: img-thumbnail
+   :alt: Terms & conditions in attachments in your quotation template?
 
-A good idea is to share more detailed or structured conditions is to
-publish on the web and to refer to that link in the terms & conditions
-of Odoo.
+.. tip::
+   This option is possible in **debug mode** in Odoo. To customize your email templates you can go
```

Comment:
I'm honestly not sure but I believe that the functional term for the debug mode is "Developer mode".. to check!